request-processor/src/application/core/utils.py

```
@@ -7,9 +7,25 @@
 
+from digital_land.specification import Specification
+
 logger = get_logger(__name__)
 
 
+def load_specification(directories):
+    if directories.S3_SPEC:
+        logger.info(f"Loading specification from S3: {directories.S3_SPEC}")
+        try:
+            return Specification(directories.S3_SPEC)
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)